### PR TITLE
hotplug_mem_negative: Update match keywords

### PR DESCRIPTION
**Pull request by huangyum**
Signed-off-by: Yumei Huang <yuhuang@redhat.com>
ID: 1961009

### CODE REVIEW

**Comment by huangyum** [comment]
(1/1) Host_RHEL.m8.u5.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.5.0.x86_64.io-github-autotest-qemu.hotplug_mem_negative.invalid_device_addr.q35: PASS (44.13 s)
@yanan-fu Pls help review, thanks.

File: qemu/tests/cfg/hotplug_mem_negative.cfg [hunk 1]
```diff
@@ -45,4 +45,4 @@
             addr_dimm_plug3 = 0x15000000000000
             keywords_plug3 = "can\'t add memory"
             addr_dimm_plug4 = 0x150800000
-            keywords_plug4 = "address range conflicts"
+            keywords_plug4 = "address range conflicts|can\'t add memory"
```
Comment (yanan-fu) [inline qemu/tests/cfg/hotplug_mem_negative.cfg:48]: IMO, `usable range for memory devices` is more suitable than `can't add memory`, how do you think ?

Comment (huangyum) [reply qemu/tests/cfg/hotplug_mem_negative.cfg:48]: I think "can't add memory" looks more like an error message than "usable range". So I prefer to keep it. Thanks.

Comment (yanan-fu) [reply qemu/tests/cfg/hotplug_mem_negative.cfg:48]: oh~   My thought it is `un-usable`......    Want to show it is the range related failure, sorry for that.

**Review by yanan-fu — APPROVED** [review]
Ack

**Pull request merged**